taxonomy-editor-frontend/src/pages/editentry/ListEntryChildren.jsx

```diff
@@ -69,14 +68,14 @@ const ListEntryChildren = ({url, setUpdateNodeChildren}) => {
         <Box>
             <Stack direction="row" alignItems="center">
                 <Typography sx={{ml: 4}} variant='h5'>Children</Typography>
-                <IconButton sx={{ml: 1, color: "#808080"}} onClick={handleOpen}>
+                <IconButton sx={{ml: 1, color: "#808080"}} onClick={handleOpenDialog}>
```

Comment:
please have this color as a `const`

Reply:
Yep, added a constant called `greyHexCode`